### ISSUE TITLE
Not detecting Steam Deck controller on NFSU

### ISSUE THREAD

**Issue by TFujiwara** [xan1242/NFSU-XtendedInput#35]
Hi, i've installed on NFSU the fix, but it doesn't detect the emulated Xbox 360 controller and nothing works in-game.
¿Is there any incompatibility with Handheld Companion?

OS: Windows 11 Pro

**Comment by xan1242** [comment]
Make sure you enable Steam Input for the game in Steam's settings, it should then work.

I've tested this firsthand with a Steam Controller and I can confirm it works.

**Comment by TFujiwara** [comment]
Same in non-steam version?

**Comment by xan1242** [comment]
Yes, you must add the game profiles to Steam (NFS Underground and 2 aren't on Steam).

**Comment by TFujiwara** [comment]
Game profile added but it does not work. ¿Maybe i'm missing something?
